docs/build-all.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# This builds the docs.cosmos.network docs using docusaurus.
+# Old documentation, which have not been migrated to docusaurus are generated with vuepress.
+COMMIT=$(git rev-parse HEAD)
+mkdir -p ~/versioned_docs  ~/versioned_sidebars
+for version in $(jq -r .[] versions.json); do
```

Comment:
Let's wait for the build docs to pass. I want to be certain `jq` is available in the GH Action. Otherwise, we need to use grep :/

Reply:
Passed :)